packages/ifpack2/test/unit_tests/CMakeLists.txt

```diff
@@ -82,6 +82,14 @@ TRIBITS_ADD_EXECUTABLE_AND_TEST(
     COMM serial mpi
   )
 
+TRIBITS_ADD_EXECUTABLE_AND_TEST(
+    DatabaseSchwarz
+    SOURCES Ifpack2_UnitTestDatabaseSchwarz.cpp Ifpack2_UnitTestMain.cpp
+    ARGS ""
+    COMM serial mpi
+    NUM_MPI_PROCS 1
```

Comment:
Since it took some work to get this working for several ranks, maybe test it also on 4 ranks?